features/admin.logs.v1/pages/logs-settings-page.tsx

```diff
@@ -0,0 +1,147 @@
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Avatar from "@oxygen-ui/react/Avatar";
+import Card from "@oxygen-ui/react/Card";
+import CardContent from "@oxygen-ui/react/CardContent";
+import Typography from "@oxygen-ui/react/Typography";
+import { DocumentIcon } from "@oxygen-ui/react-icons";
+import { AppConstants } from "@wso2is/admin.core.v1/constants/app-constants";
+import { history } from "@wso2is/admin.core.v1/helpers/history";
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import {
+    GenericIcon,
+    PageLayout
+} from "@wso2is/react-components";
+import classNames from "classnames";
+import React, { FunctionComponent, ReactElement } from "react";
+import { useTranslation } from "react-i18next";
+import "./logs.scss";
+import { LogTypeCardInterface } from "../models/log-settings-models";
+import { LogType } from "../models/remote-log-publishing";
+import "./logs-settings-page.scss";
+
+/**
+ * Proptypes for the logs page component.
+ */
+export type LogsSettingsPageInterface = IdentifiableComponentInterface;
+
+/**
+ * Logs monitoring page.
+ *
+ * @param props - {@link LogsSettingsPageInterface} Props injected to the component.
+ *
+ * @returns Logs Page {@link React.ReactElement}
+ */
+const LogsSettingsPage: FunctionComponent<LogsSettingsPageInterface> = ({
+    ["data-componentid"]: componentId = "logs-settings-page"
+}: LogsSettingsPageInterface): ReactElement => {
+
+    const { t } = useTranslation();
+
+    const logTypesCardsInfo = (): LogTypeCardInterface[] => {
+        return [
+            {
+                description: t("console:manage.features.serverConfigs.remoteLogPublishing.logTypes.audit.description"),
+                disabled: false,
+                featureStatusKey: "",
+                heading: t("console:manage.features.serverConfigs.remoteLogPublishing.logTypes.audit.name"),
+                icon: <DocumentIcon size="small" className="icon"/>,
+                identifier: LogType.AUDIT,
+                route: AppConstants.getPaths().get("LOG_SETTINGS_AUDIT")
+            },
+            {
+                description:
+                    t("console:manage.features.serverConfigs.remoteLogPublishing.logTypes.diagnostics.description"),
+                disabled: false,
+                featureStatusKey: "",
+                heading: t("console:manage.features.serverConfigs.remoteLogPublishing.logTypes.diagnostics.name"),
+                icon: <DocumentIcon size="small" className="icon"/>,
+                identifier: LogType.DIAGNOSTICS,
+                route: AppConstants.getPaths().get("LOG_SETTINGS_DIAGNOSTICS")
+            } ];
+    };
+
+    const handleBackButtonClick = (): void => {
+        history.push(AppConstants.getPaths().get("LOGS"));
+    };
+
+    return (
+        <div className="diagnostic-logs">
+            <PageLayout
```

Comment:
This div is not required here. You can pass the className directly to the `PageLayout`
And use a proper, separate class for this page. Ex: `logs-settings-page`